src/intent.html

```diff
@@ -245,9 +247,19 @@ <h3 id="mixing_intent_dictionaries">Intent Concept Dictionaries</h3>
      <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
      and also normalize
      <q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D).
-     If the speech hints are not being used
-     and the concept name is being read then each of  `-`, `_` and `.` should be
-     read as an inter-word space.</p>
+     An intent matches an entry in the core or open concept list when the normalized name, the fixity property
+     (which may be defaulted in the concept dictionary), and the number of arguments all match.
+     If a match is found, the speech hint in the list
+     should be used be used as a guide for the generation of suitable text
+     for the <a href="#intent_known_concept">known concept</a>.
+     If the intent does not match any entries, then it is treated as
+     an <a href="#intent_unknown_concept">unknown concept</a>.
+     Unknown concept names should be read as a literal after normalizing each of `-`, `_` and `.`
+     to an inter-word space.
+    Both <a href="#intent_known_concept">known concepts</a> and
+    <a href="#intent_unknown_concept">unknown concepts</a> should be read in a manner
+    consistent with any given or default arity properties.
```

Comment:
I don't think they are defined,. How about .... `consistent with the number of arguments provided in this instance of the intent` ?

Reply:
@dginev  @NSoiffer  Ah it is defined implicitly earlier:

> `because a match makes use of arity properties     (<code>prefix</code>, <code>infix</code>, etc.).`

But elsewhere these are called fixity properties rather than arity properties which seems to be a better name. I'll make an explicit commit suggestion below.